host-spec/c06-consensus.tm

```diff
@@ -302,28 +369,47 @@
 
   For any slot <math|i> in epoch <math|n> where <math|d\<less\>\<tau\>>, the
   block producer is required to produce a block. For the definitions of
-  <name|Epoch-Randomness<math|>> and <em|<name|VRF>> functions, see Algorithm
-  <reference|algo-epoch-randomness> and Section <reference|sect-vrf>
+  <name|Epoch-Randomness<math|>> and <em|<name|VRF>> functions, see Section
+  <reference|sect-epoch-randomness> and Section <reference|sect-vrf>
   respectively.
 
   <subsection|Slot Number Calculation>
 
-  It is essential for a block producer to calculate and validate the slot
-  number at a certain point in time. Slots are dividing the time continuum in
-  an overlapping interval. At a given time, the block producer should be able
-  to determine the set of slots which can be associated to a valid block
-  generated at that time. We formalize the notion of validity in the
-  following definitions:
+  It is imperative for the security of the network that each block producer
```

Comment:
This sentence is too long and hard to understand. Also, the issue is not to determine the set of slots. It is determining the slot number at a given time. It will be better to rephrase it.

Reply:
Good catch. I think this sentence is still from the time where I assumed the timestamp in a block are actually checked by every validator. Will update accordingly.